BrowseInRemoteGitRepo/BrowseInRemoteGitRepoCommand.cs

```diff
@@ -357,7 +362,7 @@ private void MenuItemCallback(object sender, EventArgs e, Action<string> process
                 gittedFilename = RunGitCommand($"ls-files \"{gittedFilename}\"", baseLocalRepoRoot)
                     .Split(newLineSeparators, StringSplitOptions.RemoveEmptyEntries)[0];
 
-                var fullUrl = baseUrl + "/blob/" + branch + "/" + gittedFilename;
+                var fullUrl = urlPattern.Replace("{baseUrl}", baseUrl).Replace("{branch}", branch).Replace("{filepath}", gittedFilename);
```

Comment:
Can `filepath` be `filePath` instead please? That would be consistent with the casing of `baseUrl`, and also a bit more readable.

Reply:
This is resolved in the latest commit.